src/scss/_variables.scss

```diff
@@ -14,24 +9,13 @@ $o-message-is-silent: true !default;
 /// @type Boolean
 $o-message-all-states-output: false !default;
 
-/// Default button class used when $o-buttons-is-silent is true
-///
-/// @type String
-$o-message-class: o-message !default;
-
-//Accepted message types
-$o-message-action-types: (action, action-bleed);
-$o-message-alert-types: (alert, alert-bleed, alert-inner);
-$o-message-notice-types: (notice, notice-bleed, notice-inner);
-
 // Alert message height
-$_o-message-action-height: 40px;
+$_o-message-spacing: 40px;
 
 //Icons
 $_o-message-alert-icon-size: 40;
 $_o-message-close-icon-size: 26;
 
-
 //Typography
 $o-message-typography-scale: 0 !default;
```

Comment:
If the user wants to configure the font scale we should probably enable that through a function/branding somehow. Might be better to keep this private until needed?